server/handler/static_files.go

```diff
@@ -0,0 +1,15 @@
+package handler
+
+import (
+	"net/http"
+	"os"
+)
+
+func FrontendStatics(w http.ResponseWriter, r *http.Request) {
+	filepath := "build" + r.URL.Path
+	if _, err := os.Stat(filepath); err == nil {
+		http.ServeFile(w, r, filepath)
+	}
```

Comment:
and here we go. empty return = mistake.

it was easy to spot in a handler with 5 lines of code.
But in with more code it's' more difficult.

That's why I'm so aggressively against of empty return when we don't have to use them.

Reply:
The problem was caused because the `return` from the original sourced dissapeared (:unamused:) when the original code was moved to the handler package.

I don't see how could it be avoided by a qualified return; example, given the following code:
```go
func () string {
    if (cond) {
        doSomething()
        return "true"
    }
    return "false"
}
```
the one above will compile too, and has the same problem.

```go
func () string {
    if (cond) {
        doSomething()
    }
    return "false"
}
```

Reply:
```go
func FrontendStatics(w http.ResponseWriter, r *http.Request) {
    http.ServeFile(w, r, func() string {
        filepath := "build" + r.URL.Path
        if _, err := os.Stat(filepath); err == nil {
            // you can't write filepath without return, it won't compile
            return filepath
        }
        return "build/index.html"
    }())
}
```

but I don't suggest to change this handler. It's small, so no need to do it more complex. But for other handlers it makes sense.

P.S. Of course you can always make a mistake, just the chance is lower with non-empty returns.